diff --git a/node_modules/ol-load-geopackage/dist/ol-load-geopackage.js b/node_modules/ol-load-geopackage/dist/ol-load-geopackage.js
index 8b6ffb7..66c6ba4 100644
--- a/node_modules/ol-load-geopackage/dist/ol-load-geopackage.js
+++ b/node_modules/ol-load-geopackage/dist/ol-load-geopackage.js
@@ -14,11 +14,12 @@ import ol_format_WKB from 'ol/format/WKB';
 
 // For reading OGC GeoPackage files, use the sql.js SQLite reader;
 // initSqlJs() will load the sql-wasm.wasm file from the current directory
-const promiseSqlWasmLoaded = initSqlJs();
-promiseSqlWasmLoaded
-    .catch(error => {
-        console.error("Error loading SQLite JS wasm binary: " + error);
-        throw error;
+const promiseSqlWasmLoaded = initSqlJs({
+  locateFile: file => '/sql-wasm.wasm'
+});
+promiseSqlWasmLoaded.catch(error => {
+    console.error("Error loading SQLite JS wasm binary: " + error);
+    throw error;
     });
 
 // -------- Public Functions --------
